__tests__/app.test.js

```diff
@@ -53,6 +53,8 @@ describe("GET article by id", ()=>{
             expect(article).toHaveProperty("created_at")
             expect(article).toHaveProperty("votes")
             expect(article).toHaveProperty("article_img_url")
+
+            expect(article).toHaveProperty("comment_count")
```

Comment:
You can be specific here - what do you expect `comment_count` `.toBe` ? 😄 We need to know the right value is being sent back on this property